src/app/app.js

```diff
@@ -146,6 +147,19 @@ angular
       })
       .when('/censusdata/create', {
         templateUrl: 'app/views/censusdata/create.html',
+        //TODO: Will think about should conroller be seperated out for each of these views.
+        controller: 'CensusDataCreateController',
```

Comment:
Sounds good. When I first looked at it, I wasn't paying enough attention that this is for the 3-part "create" functionality. I was thinking about situations where I have made separate controllers for Create and Update.